src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -871,7 +871,8 @@ private function addCacheSection(ArrayNodeDefinition $rootNode)
                     ->children()
                         ->scalarNode('prefix_seed')
                             ->info('Used to namespace cache keys when using several apps with the same shared backend')
-                            ->example('my-application-name')
+                            ->defaultValue('_%kernel.project_dir%.%kernel.container_class%')
```

Comment:
not needed, isn't it? if confirmed, this should be reverted to me.

Reply:
I think it's good to document the effective default value here. That is what the semantic configuration is for IMO. Otherwise people have to look into the implementation (CachePoolPass) to figure out what is the default which is important if you try to overwrite it. People trying to overwrite it without knowing that there is a default value at all or what the value is, are left in the dark here.